test/test_cuda.py

```diff
@@ -55,6 +56,9 @@
     TEST_LARGE_TENSOR = torch.cuda.get_device_properties(0).total_memory >= 12e9
     TEST_MEDIUM_TENSOR = torch.cuda.get_device_properties(0).total_memory >= 6e9
     TEST_BF16 = torch.cuda.is_bf16_supported()
+    TEST_GRAPH = (torch.version.cuda and int(torch.version.cuda.split(".")[0]) < 11) or (torch.version.hip and float(".".join(torch.version.hip.split(".")[0:2])) >= 5.3)
```

Comment:
Is the CUDA condition correct? `torch.version.cuda.split(".")[0]) < 11`

Reply:
@dllehr-amd Also, is this where you mentioned that the `torch.version.hip` was giving a value on a different scale than the ROCm version?

Reply:
Good catch Jithun!
  The original check was when to skip graphs, and I'm changing to when to add them.  I'll flip that over now
  
  In my case it was `TORCH_HIP_VERSION` vs `ROCM_VERSION`

Reply:
Okay, this is what I'm doing in the other code.  Essentially ignore TORCH_HIP_VERSION if use_rocm is specified.

In this case we can still leave it yes?  calling the pythonic method to get driver versions to determine if we can run tests?